### PR TITLE
fix: slash runtime path

### PR DESCRIPTION
**Pull request by markthree**
close [nuxt/issues/25865](https://github.com/nuxt/nuxt/issues/25865)

Although this is a temporary solution, the better approach would definitely be to fix it in Nitro. However, I want to test whether it would introduce any break changes in Nitro, and then I will attempt to make a pull request 🤣

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -56,13 +60,13 @@ async function nitroModule(nitro: Nitro) {
   // Make sure runtime is transpiled
   nitro.options.externals.inline = nitro.options.externals.inline || [];
   nitro.options.externals.inline.push(
-    fileURLToPath(new URL("runtime/", import.meta.url)),
+    slash(fileURLToPath(new URL("runtime/", import.meta.url))),
```
Comment (pi0) [inline src/index.ts:63]: We might use `fileURLToPath` from `mlly` 1.6 which accepts URL and normalizes slash now ;)

Comment (markthree) [reply src/index.ts:63]: > We might use `fileURLToPath` from `mlly` 1.6 which accepts URL and normalizes slash now ;)

Nice, I'll try it now.

Comment (markthree) [reply src/index.ts]: done

**Review by pi0 — APPROVED** [review]
Thanks!

**Pull request merged**